content/en/network_monitoring/devices/data.md

```diff
@@ -8,6 +8,7 @@ aliases:
 ## Metrics
 
 Network Device Monitoring submits specified metrics under the `snmp.*` namespace. The metrics collected are determined by the `[configured profile]`.
+If metrics you want is not on the following list, you can search OID and its name from [Global OID reference database](http://oidref.com/) and add your profiles.
```

Comment:
Add reference link to the bottom of the page:
`[1]: http://oidref.com`
```suggestion
If the metrics you want are not in the following list, search for the OID and name from the [Global OID reference database][1] to add to your profiles.
```

Reply:
Thank you for your review!
Fixed https://github.com/DataDog/documentation/pull/12664/commits/b00cbf9aade9a536d1d745b5505f1e078e55b412